tests/integration/test_live_darwin_api.py

```
@@ -0,0 +1,263 @@
+                    break
+
+    def test_delayed_service_format(self, live_darwin_session):
+        """Verify format of delayed services matches our mocks"""
+        board = live_darwin_session.get_station_board('PAD', rows=50)
+
+        if hasattr(board, 'train_services') and board.train_services:
+            for service in board.train_services:
+                # Look for actual time (HH:MM format) instead of "On time"
+                if hasattr(service, 'etd') and ':' in service.etd and \
+                   service.etd != service.std:
+                    print(f"\nDelayed service found:")
```

Ruff (0.14.14)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)